framework/moose.mk

```diff
@@ -168,7 +168,7 @@ moose_all_header_dir := $(all_header_dir)
 define all_header_dir_rule
 $(1):
 	@echo Rebuilding symlinks in $$@
-	@$$(shell mkdir -p $$@)
```

Comment:
@permcody did `@mkdir` not exist back in the day?

Reply:
Ha, It probably did. I think this is Derek’s line though.